ansible/roles/software/ckan/security-fix/tasks/main.yml

```diff
@@ -10,3 +10,21 @@
   with_items:
     - /etc/ckan
     - "{{ project_source_new_code_path }}"
+
+- name: Fix 'Content-Cache' header
+  replace:
+    dest: "{{ project_source_new_code_path }}/src/ckan/ckan/lib/base.py"
+    regexp: ' = "private"'
```

Comment:
The benefit of lineinfile is that it will add required line even if it doesn't find the pattern. 
There is also [blockinfile](http://docs.ansible.com/ansible/blockinfile_module.html) module for multiple lines.